src/org/tn5250j/keyboard/KeyMapper.java

```diff
@@ -157,7 +157,6 @@ public static void init() {
 
 	private static boolean loadKeyStrokes(Properties keystrokes) {
 
-		keystrokes = ConfigureFactory.getInstance().getProperties(ConfigureFactory.KEYMAP);
```

Comment:
When removing this, the loadKeyStrokes() makes no sense anymore, because it's not loading, but just checking if size is OK.
Thus I propose to rename the method to 'containsProperties()